apis/elasticache/v1alpha1/custom_types.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+
+// CustomUserParameters contains the addtionals fields for UserParameters
+type CustomUserParameters struct {
+	PasswordSecretRef xpv1.SecretKeySelector `json:"passwordSecretRef,omitempty"`
```

Comment:
It seems like the API takes an array of passwords, so I think this should be an array of secret key selectors.

Reply:
i will think about more then one password - in general we have PasswordSecretRef for array on no other ressource if i am correct

hint:
_Warning! Both passwords will be updated. If you are only updating one password, please enter the second one as well in order to retain it._

Reply:
@muvaf can we go with followup PR for Passwords and merge this PR ? i will focus on followup to have array of passwords

Reply:
@muvaf can we go with followup PR for Passwords and merge this PR ? i will focus on followup to have array of passwords

Reply:
fixed!